index.ts

```diff
@@ -0,0 +1,18 @@
+interface IEventEmitter<T = any> {
```

Comment:
Can't we reuse the typings from Node here?

Reply:
Yes - indeed that would align with this PR that was just made https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58821/files

Reply:
I was trying to add some level of *actual* type safety to the type parameter I added to `arrayifyStream` - rather than just making it the pseudo type safety that it really is. But this wasn't really doing the trick anyway